install_travis_dependencies.sh

```diff
@@ -16,6 +16,7 @@ sudo apt-get install -y --force-yes gradle
 if [[ $languages == *"python"* ]]; then
   sudo apt-get install -y --force-yes python-dev python-pip
   sudo pip install --upgrade pip wheel virtualenv virtualenvwrapper --quiet
+  sudo pip install -r requirements.txt
```

Comment:
This installs virtualenv and then install the dependencies without using it. Travis spawn a new instance every time, we shouldn't need to use a package version manager.